Feature/poco lista

fernandohre opened the pull request:
Adequando ao modelo descritivo de desenvolvimento.
Fiz o commit e apareceu o nome de outra pessoa por conta de configuração dessa máquina

Commit:
adição de resposta objetiva e de enum para tipo de questão. tag para identificar uma questao

Commit:
persistencia da questao discursiva de acordo com o modelo de dados

saulocalixto committed:
Revisões, criando métodos genéricos para api.

saulocalixto committed:
Criando objeto de filtro de questão.

saulocalixto committed:
Acrescentando o filtro na consulta.

saulocalixto committed:
Voltando testes comentados.

saulocalixto committed:
Modificando Consulte para retornar lista de objeto.

gustavo-b committed:
Autenticação e autorização implementada

gustavo-b committed:
Merge pull request #3 from UFG-ES-LISTELAB/feature/filtro

Feature/filtro

gustavo-b committed:
Merging develop into feature e criando Consulta de um item apenas do banco

gustavo-b committed:
Adaptando serviço para novo repositório

saulocalixto committed:
Merge pull request #4 from UFG-ES-LISTELAB/feature/Autenticacao_e_Autorizacao

Autenticação e autorização implementada

gustavo-b committed:
Restaurando App.config

saulocalixto committed:
Merge pull request #5 from UFG-ES-LISTELAB/fix/App.config

Restaurando App.config

Commit:
Merge remote-tracking branch 'origin/dev' into feature/poco-questao

Commit:
Auto stash before merge of "feature/poco-questao" and "origin/dev"

Commit:
Implementação para questão objetiva

saulocalixto committed:
Merge pull request #6 from UFG-ES-LISTELAB/feature/poco-questao

Feature/poco questao

Commit:
descomentando regras de cadastro

saulocalixto committed:
Merge pull request #7 from UFG-ES-LISTELAB/feature/poco-questao

descomentando regras de cadastro

Commit:
adicionando usuário em questão

saulocalixto committed:
Merge pull request #8 from UFG-ES-LISTELAB/feature/poco-questao

adicionando usuário em questão

Commit:
Implementação poco-lista

FlorentinoFilho committed:
adequando implementação ao modelo descritivo de desenvolvimento

saulocalixto reviewed (changes requested):
O pullrequest foi feito para o branche errado. Ele deve ser feito para o dev, não o master.

saulocalixto committed:
Ajustes em classes poco

Colocando comentário nas classes, além de ajustar o objeto questão para ter somente o email do usuário.

saulocalixto commented:
Ele foi feito para o branche errado.
